### ISSUE TITLE
RemovedInDjango110Warning

### ISSUE THREAD

**Issue by vakorol** [divio/django-cms#5401]
It seems that not all deprecations have been fixed in Django CMS 3.3. I get the following:

```
Environment:

Request Method: GET
Request URL: http://127.0.0.1:20230/pages/?edit&language=en-us

Django Version: 1.8.13
Python Version: 3.4.3
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'rest_framework',
 'bootstrap3',
 'djkombu',
 'lib',
 'user',
 'dashboard',
 'wizard',
 'structures',
 'datasets',
 'extresources',
 'tasks',
 'potentials',
 'projects',
 'programs',
 'notifications',
 'testing',
 'filemanager',
 'django.contrib.sites',
 'djangocms_text_ckeditor',
 'cms',
 'treebeard',
 'menus',
 'sekizai',
 'djangocms_admin_style')
Installed Middleware:
('cms.middleware.utils.ApphookReloadMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.security.SecurityMiddleware',
 'viking.middleware.LoginRequiredMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'cms.middleware.user.CurrentUserMiddleware',
 'cms.middleware.page.CurrentPageMiddleware',
 'cms.middleware.toolbar.ToolbarMiddleware',
 'cms.middleware.language.LanguageCookieMiddleware')


Template error:
In template /usr/local/lib/python3.4/dist-packages/cms/templates/cms/toolbar/placeholder.html, error at line 24
   render() must be called with a dict, not a Context.

   14 :     plugin_language: '{{ language }}',
   15 :     plugin_restriction: [{% for module in allowed_plugins %}"{{ module }}"{% if not forloop.last %}, {% endif %}{% endfor %}],
   16 :     addPluginHelpTitle: '{% trans "Add plugin to placeholder" %} "{{ placeholder.get_label|escapejs }}"',
   17 :     urls: {
   18 :         add_plugin: '{{ placeholder.get_add_url }}',
   19 :         copy_plugin: '{{ placeholder.get_copy_url }}'
   20 :     }
   21 : }]);
   22 : </script>
   23 : <script id="cms-plugin-child-classes-{{ placeholder.pk|unlocalize }}" type="text/cms-template">
   24 :      {% include request.toolbar.drag_item_menu_template %} 
   25 : </script>
   26 : {% endlanguage %}
   27 : {% endaddtoblock %}
   28 : 

Traceback:
File "/usr/local/lib/python3.4/dist-packages/django/core/handlers/base.py" in get_response
  164.                 response = response.render()
File "/usr/local/lib/python3.4/dist-packages/django/template/response.py" in render
  158.             self.content = self.rendered_content
File "/usr/local/lib/python3.4/dist-packages/django/template/response.py" in rendered_content
  135.         content = template.render(context, self._request)
File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py" in render
  74.         return self.template.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  210.                     return self._render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/loader_tags.py" in render
  135.         return compiled_parent._render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/classytags/core.py" in render
  146.         return self.render_tag(context, **kwargs)
File "/usr/local/lib/python3.4/dist-packages/sekizai/templatetags/sekizai_tags.py" in render_tag
  83.         rendered_contents = nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/classytags/core.py" in render
  146.         return self.render_tag(context, **kwargs)
File "/usr/local/lib/python3.4/dist-packages/cms/templatetags/cms_tags.py" in render_tag
  631.         rendered_contents = nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/loader_tags.py" in render
  65.                 result = block.nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/classytags/core.py" in render
  146.         return self.render_tag(context, **kwargs)
File "/usr/local/lib/python3.4/dist-packages/cms/templatetags/cms_tags.py" in render_tag
  304.             content = get_placeholder_content(context, request, page, name, inherit, nodelist)
File "/usr/local/lib/python3.4/dist-packages/cms/templatetags/cms_tags.py" in get_placeholder_content
  243.     return render_placeholder(placeholder, context, name, default=default)
File "/usr/local/lib/python3.4/dist-packages/cms/plugin_rendering.py" in render_placeholder
  196.         toolbar_content = mark_safe(render_placeholder_toolbar(placeholder, context, name_fallback, save_language))
File "/usr/local/lib/python3.4/dist-packages/cms/plugin_rendering.py" in render_placeholder_toolbar
  241.     toolbar = render_to_string("cms/toolbar/placeholder.html", flatten_context(context))
File "/usr/local/lib/python3.4/dist-packages/django/template/loader.py" in render_to_string
  99.         return template.render(context, request)
File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py" in render
  74.         return self.template.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  210.                     return self._render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/sekizai/templatetags/sekizai_tags.py" in render
  66.             return super(SekizaiTag, self).render(context)
File "/usr/local/lib/python3.4/dist-packages/classytags/core.py" in render
  146.         return self.render_tag(context, **kwargs)
File "/usr/local/lib/python3.4/dist-packages/sekizai/templatetags/sekizai_tags.py" in render_tag
  145.         rendered_contents = nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/templatetags/i18n.py" in render
  182.             output = self.nodelist.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/local/lib/python3.4/dist-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/loader_tags.py" in render
  159.                 return template.render(context)
File "/usr/local/lib/python3.4/dist-packages/django/template/backends/django.py" in render
  69.                 RemovedInDjango110Warning, stacklevel=2)

Exception Type: RemovedInDjango110Warning at /pages/
Exception Value: render() must be called with a dict, not a Context.
```

I created a similar ticket #4886 in January, 2016. In our development environment, deprecation warnings are turned into errors, so this issue is actually blocking us from using Django CMS in our project. I could've ignored the warning using `warnings.filterwarnings`, but since it's raised in django core, i can't filter the warning by the app name.

**Comment by czpython** [comment]
Thanks for reporting. There's an ongoing effort to fix these warnings in #5434

**Comment by czpython** [comment]
Fixed on 3.4.0 (currently in rc)

**Comment by czpython** [comment]
Django 1.10 is not officially supported yet as we're waiting on django-treebeard to be released with official Django 1.10 support.
